src/postgres/commands/enterPostgresCredentials.ts

```diff
@@ -32,7 +32,22 @@ export async function enterPostgresCredentials(context: IActionContext, treeItem
         validateInput: (value: string) => { return (value && value.length) ? undefined : localize('passwordCannotBeEmpty', 'Password cannot be empty.'); }
     });
 
-    const server: Server = nonNullProp(treeItem, 'server');
-    await setPostgresCredentials(username, password, nonNullProp(server, 'id'));
+    const serverName: string = nonNullProp(treeItem.server, 'name');
+    const id: string = nonNullProp(treeItem.server, 'id');
+
+    const progressMessage: string = localize('setupCredentialsMessage', 'Setting up Credentials for server "{0}"...', serverName);
```

Comment:
nit: small capitalization fix
```suggestion
    const progressMessage: string = localize('setupCredentialsMessage', 'Setting up credentials for server "{0}"...', serverName);
```